### DISCUSSION TITLE
Q: How to Stop Getting 403 Error After Upgrading Waline from V2 to V3

### DISCUSSION THREAD

**Discussion by ilovefreesw** [walinejs/waline#2331] [Q&A]
### Describe the bug | 问题描述

I upgraded waline from v2 to v3 by changing the imports in head and footer both and redeployed the sever on Vercel. Now, when someone comments, they get "Forbidden" alert after submitting the comment.

In the vercel logs, these are errors:

![image](https://github.com/walinejs/waline/assets/77317006/21e4a961-a122-41ec-8474-cf4e6e0b5e47)

Backend works fine. I can log in and see the older approved comments. But they are not displayed on the site.

In the browser console, this is the error.

![image](https://github.com/walinejs/waline/assets/77317006/32d7c092-9d64-4992-b96e-ca040dda7c56)

### Website URL | 问题网站

https://ilovefreesoftware.com

### Where your waline deploy? | 服务部署在哪里？

Vercel (Default)

### Where your comment data store? | 数据存储在哪里？

MySQL

**Comment by lizheming** [comment]
It should be because the SECURE_DOMAIN environment variable is configured. You can check whether the variable contains the domain name `www.ilovefreesoftware.com`, or delete the SECURE_DOMAIN environment variable.

**Reply by ilovefreesw** [reply]
I had this configured ... before ..... just forgot the domain with "www" was not there .. my mistake... sorry for the inconvenience.
